pkg/kubelet/cadvisor/cadvisor_linux_docker.go

```diff
@@ -0,0 +1,25 @@
+// +build linux,!dockerless
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cadvisor
+
+import (
+	// We only want to perform this docker specific cadvisor init when we are not
+	// using the `dockerless` build tag.
+	_ "github.com/google/cadvisor/container/docker/install"
```

Comment:
just noting: we actually do this for both containerd and crio as well for as long as cadvisor is still enabled in kubelet.

Reply:
this should maybe not be in the dockerless tag then...?

Reply:
Just want to be sure I understand - we still do the `docker/install`? Can you share more around why that's necessary in situations where we aren't using docker?